src/main/java/de/retest/web/ScreenshotProvider.java

```diff
@@ -49,7 +49,7 @@ private static BufferedImage shootViewportOnly( final WebDriver driver ) {
 	 * @return The wrapped driver if present, otherwise the given driver.
 	 */
 	private static WebDriver unwrap( final WebDriver driver ) {
-		return driver instanceof UnbreakableDriver ? ((UnbreakableDriver) driver).getWrapped() : driver;
+		return driver instanceof UnbreakableDriver ? ((UnbreakableDriver) driver).getWrappedDriver() : driver;
```

Comment:
To avoid further issues with Shutterbug, I would suggest to now use the interface you've discovered:

```suggestion
		return driver instanceof WrapsDriver ? ((WrapsDriver) driver).getWrappedDriver() : driver;
```